Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -215,6 +215,8 @@ open class TabToolbarHelper: NSObject {
         switch middleButtonState {
         case .search:
             return
+        case .home:
+            return
```

Comment:
nit: a bit more concise
```suggestion
        case .search, .home:
            return
```